src/DotNetLightning.Core/Channel/Channel.fs

```diff
@@ -16,25 +16,33 @@ open System
 type ProvideFundingTx = IDestination * Money * FeeRatePerKw -> Result<FinalizedTx * TxOutIndex, string> 
 type Channel = {
     Config: ChannelConfig
-    KeysRepository: IKeysRepository
+    ChannelPrivKeys: ChannelPrivKeys
     FeeEstimator: IFeeEstimator
     FundingTxProvider:ProvideFundingTx
     RemoteNodeId: NodeId
-    LocalNodeSecret: Key
+    NodeSecret: NodeSecret
     State: ChannelState
     Network: Network
-    Secp256k1Context: ISecp256k1
  }
         with
-        static member Create(config, keysRepo, feeEstimator, localNodeSecret, fundingTxProvider, n, remoteNodeId) =
+        static member Create (
+                config: ChannelConfig,
+                nodeMasterPrivKey: NodeMasterPrivKey,
```

Comment:
I think you're introducing indentation warnings here too

Reply:
the way to fix I think is this way: instead of

```
Create (
    foo,
    bar
```

this way:

```
Create (foo,
             bar,
```

Reply:
Fixed.